Integration tests leave root-owned files in target

- Run integration tesdts with `mvn clean test -Pintegration` - they pass
- run `mvn clean` - it fails with
```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-clean-plugin:2.5:clean (default-clean) on project cartridge-driver: Failed to clean project: Failed to delete /home/isopov/java/workspace/cartridge-java/target/test-classes/cartridge/.rocks/bin/stateboard
```
I have docker accessible to my user and run integration under it not under root.

--- a/.github/workflows/ubuntu-master.yml
+++ b/.github/workflows/ubuntu-master.yml
@@ -29,4 +29,4 @@ jobs:
     - name: Build with Maven
       run: mvn -B verify --file pom.xml
     - name: Run integration tests
-      run: mvn -B test -P integration --file pom.xml
+      run: mvn -B test -P integration --file pom.xml -DdockerfileUseRoot=true

--- a/pom.xml
+++ b/pom.xml
@@ -135,7 +135,7 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-surefire-plugin</artifactId>
-                <version>3.0.0-M4</version>
+                <version>3.0.0-M5</version>
                 <configuration>
                     <includes>
                         <include>**/*Test.java</include>
@@ -222,7 +222,7 @@
         <dependency>
             <groupId>org.testcontainers</groupId>
             <artifactId>testcontainers</artifactId>
-            <version>1.15.0-rc2</version>
+            <version>1.15.0</version>
             <scope>test</scope>
         </dependency>
         <dependency>
@@ -234,7 +234,7 @@
         <dependency>
             <groupId>io.tarantool</groupId>
             <artifactId>testcontainers-java-tarantool</artifactId>
-            <version>0.2.0</version>
+            <version>0.2.3</version>
             <scope>test</scope>
         </dependency>
     </dependencies>

--- a/src/main/java/io/tarantool/driver/AbstractTarantoolClient.java
+++ b/src/main/java/io/tarantool/driver/AbstractTarantoolClient.java
@@ -29,6 +29,7 @@ import io.tarantool.driver.protocol.requests.TarantoolEvalRequest;
 import io.tarantool.driver.utils.Assert;
 import org.msgpack.value.ArrayValue;
 
+import java.util.Arrays;
 import java.util.Collections;
 import java.util.List;
 import java.util.Optional;
@@ -154,6 +155,12 @@ public abstract class AbstractTarantoolClient implements TarantoolClient {
     @Override
     public CompletableFuture<List<?>> call(String functionName) throws TarantoolClientException {
         return call(functionName, Collections.emptyList());
+    }
+
+    @Override
+    public CompletableFuture<List<?>> call(String functionName, Object... arguments)
+            throws TarantoolClientException {
+        return call(functionName, Arrays.asList(arguments));
     }
 
     @Override

--- a/src/main/java/io/tarantool/driver/ProxyTarantoolClient.java
+++ b/src/main/java/io/tarantool/driver/ProxyTarantoolClient.java
@@ -20,6 +20,7 @@ import io.tarantool.driver.metadata.ProxyTarantoolMetadata;
 import io.tarantool.driver.utils.Assert;
 import org.msgpack.value.ArrayValue;
 
+import java.util.Arrays;
 import java.util.List;
 import java.util.Optional;
 import java.util.concurrent.CompletableFuture;
@@ -118,6 +119,12 @@ public class ProxyTarantoolClient implements TarantoolClient, ProxyOperationsMap
     @Override
     public CompletableFuture<List<?>> call(String functionName) throws TarantoolClientException {
         return client.call(functionName);
+    }
+
+    @Override
+    public CompletableFuture<List<?>> call(String functionName, Object... arguments)
+            throws TarantoolClientException {
+        return client.call(functionName, Arrays.asList(arguments));
     }
 
     @Override

--- a/src/main/java/io/tarantool/driver/api/TarantoolCallOperations.java
+++ b/src/main/java/io/tarantool/driver/api/TarantoolCallOperations.java
@@ -32,6 +32,18 @@ public interface TarantoolCallOperations {
      * used for converting the result values from MessagePack entities to objects.
      *
      * @param functionName function name, must not be null or empty
+     * @param arguments    vararg array of function arguments. The object mapper specified in the client configuration
+     *                     will be used for arguments conversion to MessagePack entities
+     * @return some result
+     * @throws TarantoolClientException if the client is not connected
+     */
+    CompletableFuture<List<?>> call(String functionName, Object... arguments) throws TarantoolClientException;
+
+    /**
+     * Execute a function defined on Tarantool instance, The value mapper specified in the client configuration will be
+     * used for converting the result values from MessagePack entities to objects.
+     *
+     * @param functionName function name, must not be null or empty
      * @param arguments    list of function arguments. The object mapper specified in the client configuration
      *                     will be used for arguments conversion to MessagePack entities
      * @return some result

--- a/src/test/java/io/tarantool/driver/integration/ProxyTarantoolClientIT.java
+++ b/src/test/java/io/tarantool/driver/integration/ProxyTarantoolClientIT.java
@@ -60,7 +60,11 @@ public class ProxyTarantoolClientIT extends SharedCartridgeContainer {
         USER_NAME = container.getUsername();
         PASSWORD = container.getPassword();
         initClient();
-        client.space(TEST_SPACE_NAME).delete(Conditions.any()).get();
+        truncateSpace(TEST_SPACE_NAME);
+    }
+
+    private static void truncateSpace(String spaceName) {
+        client.call("truncate_space", spaceName);
     }
 
     private static TarantoolClusterAddressProvider getClusterAddressProvider() {

--- a/src/test/resources/cartridge/app/roles/api_router.lua
+++ b/src/test/resources/cartridge/app/roles/api_router.lua
@@ -1,4 +1,6 @@
 local vshard = require('vshard')
+local cartridge_pool = require('cartridge.pool')
+local cartridge_rpc = require('cartridge.rpc')
 
 -- function to get cluster schema
 local function crud_get_schema()
@@ -37,10 +39,15 @@ local function crud_get_schema()
     return uniq_spaces
 end
 
+local function truncate_space(space_name)
+    local storages = cartridge_rpc.get_candidates('app.roles.api_storage')
+    cartridge_pool.map_call('box.schema.space[' .. space_name .. ']:truncate', nil, {uri_list = storages})
+end
 
 local function init(opts)
 
     rawset(_G, 'crud_get_schema', crud_get_schema)
+    rawset(_G, 'truncate_space', truncate_space)
 
     return true
 end

